front/src/Components/PDFViewer.tsx

```diff
@@ -55,27 +60,87 @@ export const LoadingComponent = () =>
     <FontAwesomeIcon icon={faSpinner} size="6x" spin />
   </div>;
 
+const glossariumAPI = new GlossariumAPI();
+
 const PDFViewer = (props: PDFViewerProps & RouteComponentProps) => {
-  const [pageNumber, setPageNumber] = React.useState<number>(1);
+  const [pageNumber, setPageNumber] = React.useState<number>(0);
+  const [wordhoardIDs, setWordhoardIDs] = React.useState<string[]>([]);
   const [docRef, setDocRef] = React.useState<any>(null);
   const [numPages, setNumPages] = React.useState<number>(0);
   const [maxWidth] = React.useState<number>(calcMaxWidth(window.innerWidth));
   const [showButtons, setShowButtons] = React.useState<boolean>(false);
   const drawer = React.useContext(SideDrawerContext);
   const pdfWrapper = React.createRef<HTMLInputElement>();
 
+  const glossIsOpen =
+    myPersistedState<boolean>("orisearch.pdfviewer.glossariumOpened", false);
+
+  const setDocumentSectionAnnotations =
+    usePersistedState<any>("orisearch.pdfviewer.documentSectionAnnotations", [])[1];
+
+  const documentID = myPersistedState<string>("orisearch.pdfviewer.documentID", "");
```

Comment:
Not sure whether it's the right thing to use the `persistedState` here, since it persists in between sessions. Why use that instead of `useState`?

Reply:
Used it because we need share state between components. If you have a suggestion on how to share state between components, let me know. 

SessionStorage perhaps, using Redux seems like overkill for sharing a couple values between components.

Reply:
done